src/components/atoms/BalanceGameButton/BalanceGameButton.style.ts

```diff
@@ -11,31 +11,27 @@ export const buttonWrapStyle = (option: 'A' | 'B') =>
     flexDirection: 'column',
     alignItems: 'center',
     width: '561px',
+    minHeight: '488px',
+    flexGrow: 1,
     backgroundColor: 'transparent',
     borderRadius: option === 'A' ? '20px 0 0 20px' : '0 20px 20px 0',
+    overflow: 'hidden',
     '&:hover': {
+      zIndex: option === 'A' ? 100 : 'auto',
```

Comment:
```suggestion
    overflow: 'hidden',
    
    '&:hover': {
      zIndex: option === 'A' ? 100 : 'auto',
```
처럼 줄바꿈 표시를 해주면 가독성이 더 좋아질 것 같습니다!

Reply:
[65c2746](https://github.com/CHZZK-Study/Balance-Talk-Frontend/pull/200/commits/65c2746f8b933b366b08de9ff7d296aaeb1f5d73) 에서 수정했습니다!